routers/web/repo/pull_review.go

```diff
@@ -85,6 +93,7 @@ func CreateCodeComment(ctx *context.Context) {
 		!form.SingleReview,
 		form.Reply,
 		form.LatestCommitID,
+		attachments,
```

Comment:
```suggestion
		form.Files,
```

Reply:
It is checked in these other locations:
* https://github.com/go-gitea/gitea/blob/d3982bcd814bac93e3cbce1c7eb749b17e413fbd/routers/web/repo/issue.go#L1215-L1217
* https://github.com/go-gitea/gitea/blob/d3982bcd814bac93e3cbce1c7eb749b17e413fbd/routers/web/repo/issue.go#L2974-L2977
* https://github.com/go-gitea/gitea/blob/d3982bcd814bac93e3cbce1c7eb749b17e413fbd/routers/web/repo/pull_review.go#L234-L237
* https://github.com/go-gitea/gitea/blob/d3982bcd814bac93e3cbce1c7eb749b17e413fbd/routers/web/repo/release.go#L411-L414

So I did the same here as I assumed there must be a reason for that check.